scala-package/core/src/main/scala/org/apache/mxnet/IO.scala

```diff
@@ -280,7 +333,10 @@ abstract class DataIter extends Iterator[DataBatch] {
    */
   @throws(classOf[NoSuchElementException])
   def next(): DataBatch = {
-    new DataBatch(getData(), getLabel(), getIndex(), getPad())
+    new DataBatch(getData(), getLabel(), getIndex(), getPad(),
+      null, null, null,
```

Comment:
please make these parameters as Options and don't pass nulls.

Reply:
This will break bcc I think, not sure about the influence